### PR TITLE
Auto-generate all-in-one-supplemental dimensions

### PR DESCRIPTION
**Pull request by daniel-thom**
Paired with https://github.com/dsgrid/dsgrid/pull/125

### CODE REVIEW

File: test_efs/dsgrid_project/project.toml [hunk 1]
```diff
@@ -15,6 +15,7 @@ model_sector = "cld"
 version = "1.0.0"
 
 [dimensions]
+all_in_one_supplemental_dimensions = ["sector", "subsector"]
```
Comment (daniel-thom) [inline test_efs/dsgrid_project/project.toml:18]: This is the one thing that is new in this PR.

File: test_aeo/dsgrid_project/datasets/benchmark/Commercial_End_Use_Growth_Factors/dataset.toml [hunk 2]
```diff
@@ -98,6 +105,7 @@ Weather years, 2013-2050, trivial, single record
 [[dimensions]] # already exists
 type = "model_year"
 name = "Model_Years-trivial-2013_to_2050"
+display_name = "None"
```
Comment (elainethale) [inline test_aeo/dsgrid_project/datasets/benchmark/Commercial_End_Use_Growth_Factors/dataset.toml:108]: So users are supposed to put "None" in for all trivial dimensions?

File: test_efs/dsgrid_project/datasets/sector_models/comstock/dimensions.toml [hunk 3]
```diff
@@ -15,6 +16,7 @@ file = "dimensions/model_years.csv"
 module = "dsgrid.dimension.standard"
 name = "Model Year 2012"
 type = "model_year"
+display_name = "2012"
```
Comment (elainethale) [inline test_efs/dsgrid_project/datasets/sector_models/comstock/dimensions.toml:19]: I see not always--trivial dimensions might also list their actual record values ...

**Review by elainethale — APPROVED** [review]
I think you can ignore my comments ... you aren't trying to address trivial dimensions with this PR

File: test_aeo/dsgrid_project/datasets/benchmark/Commercial_End_Use_Growth_Factors/dataset.toml (hunk 2 above)
Comment (daniel-thom) [reply test_aeo/dsgrid_project/datasets/benchmark/Commercial_End_Use_Growth_Factors/dataset.toml:108]: Uhh...I think you made that commit.

**Pull request merged**